.gitignore

```diff
@@ -1,5 +1,6 @@
 *.pyc
 *.egg-info/
+*.swp
```

Comment:
The vim editor makes swp files when you're editing a file and haven't saved yet, it's a buffer for the editor. sometimes it peppers these files around, so I want to make sure I don't accidentally check them in as commits.